contrib/local-environment/Makefile

```
@@ -14,6 +14,14 @@ alpha-config-up:
 alpha-config-%:
 	docker compose -f docker-compose.yaml -f docker-compose-alpha-config.yaml $*
 
+.PHONY: multi-tenancy-up
```

can we rename this to:

```suggestion
.PHONY: multi-provider-up
```